src/ol/render/webgl/webglreplay.js

```diff
@@ -1240,6 +1243,38 @@ ol.render.webgl.LineStringReplay.prototype.isClosed_ = function(flatCoordinates,
   return false;
 };
 
+/**
+ * Check if the linestring can be drawn (i. e. valid).
+ * @param {Array.<number>} flatCoordinates Flat coordinates.
+ * @param {number} offset Offset.
+ * @param {number} end End.
+ * @param {number} stride Stride.
+ * @param {boolean} closed The linestring is a boundary.
+ * @return {boolean} The linestring can be drawn.
+ * @private
+ */
+ol.render.webgl.LineStringReplay.prototype.isValid_ = function(flatCoordinates, offset, end, stride,
```

Comment:
Same thoughts as on `isClosed_`. The function is operating on flat LineStrings, and thus it might go in the `ol.geom.flat` namespace.